### PR TITLE
Lesson 1 8

### PR DESCRIPTION
**Pull request by Victoria240**
Data Fetching, Re-Fetching, and Memoized Handlers - Updated

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -1,63 +1,120 @@
...
+    // Construct the URL for the API
+    const url = `https://api.airtable.com/v0/${process.env.REACT_APP_AIRTABLE_BASE_ID}/${process.env.REACT_APP_TABLE_NAME}`;
+
+
+    try {
+      const response = await fetch(url, options);
+
+
+      if (!response.ok) {
+        throw new Error(`Error: ${response.status}`);
```
Comment (ezbc) [inline src/App.js:30]: This is what the homework asked for and you aced it!

For future reference, this display an error like `Error: 400`. `response.status` is the HTTP status code which may not be super helpful for debugging. You could log the response in the console as well to provide more information for debugging when errors occur like:

```
console.log('Airtable error response:')
console.log(response)
throw new Error(`Error: ${response.status}`);
```

File: src/App.js [hunk 2]
```diff
@@ -1,63 +1,120 @@
...
 
+      // Parse the response JSON
+      const data = await response.json();
 
 
-    function removeTodo(id) {
-      // Filter the todoList array to exclude the todo item with the specified id
-      const updatedTodoList = todoList.filter((todo) => todo.id !== id);
+      // Print the fetched data to observe Airtable's API response
+      console.log('Fetched data:', data);
```
Comment (ezbc) [inline src/App.js:39]: I think you can remove this now

**Review by ezbc — APPROVED** [review]
Awesome job Victoria! This worked like a charm. I added one teeny suggestion for future reference.

File: src/App.js (hunk 1 above)
Comment (Victoria240) [reply src/App.js:30]: Thank you, Elijah!

**Pull request merged**